desks-web-app/src/components/booking-components/WeekOverviewWindow.vue

```diff
@@ -1,20 +1,56 @@
 <template>
 
   <v-row>
-    <template v-for="n in 5" :key="n">
-      <v-col>
+    <template v-for="n in [0,1,2,3,4,5,6]" :key="n">
+      <v-col  :class="{days: true, monday: calculateNthNextDay(n).isoWeekday() === 1}" v-if="this.bookingDefaults && calculateNthNextDay(n).isoWeekday() <= 5">
```

Comment:
`v-if="this.bookingDefaults && calculateNthNextDay(n).isoWeekday() <= 5`
that filter looks a bit janky. It might be easier to understand if you. Change the `calculateNthNextDay` method to calculate the nth next week day.

Reply:
`calculateNthNextDay(n).isoWeekday() === 1` please make a method out of that because it is not obvious that it is the check if it is a monday.